cpp/test/Slice/errorDetection/WrongProxyType.ice

```diff
@@ -8,12 +8,12 @@ module Test
     interface I
     {
         Seq* f1();
-        void f2(Seq*);
-        void f3(out Seq*);
+        void f2(Seq* s);
+        void f3(out Seq* s);
 
         Dict* f4();
-        void f5(Dict*);
-        void f6(out Dict*);
+        void f5(Dict* s);
+        void f6(out Dict* s);
```

Comment:
All these parameters were missing an identifier.
But because of a type-resolution-error (putting '*' after a sequence is bogus),
this "missing identifier" error was being skipped over.